### PR TITLE
LF-2478/pseudo user conversion ui change

### PR DESCRIPTION
**Pull request by seinnlee**
Ticket [LF-2478](https://lite-farm.atlassian.net/browse/LF-2478)

**To Test:**
1. Create a pseudo user (Go to people tab of the farm and invite a user without an email)
2. Click the pseudo user and click "Convert this worker to a user with account"
3. You should see all the fields same as regular invitation UI
4. Fill out the form and invite the user
5. Check in the users table from the database that the information has been correctly updated

### CODE REVIEW

File: packages/api/src/controllers/userFarmController.js [hunk 1]
```diff
@@ -500,7 +500,7 @@ const userFarmController = {
   patchPseudoUserEmail() {
```
Comment (fsayyid) [inline packages/api/src/controllers/userFarmController.js:500]: Refactor this name to reflect that not only the email is being patched. Maybe `upgradePseudoUser` or something.

File: packages/api/src/controllers/userFarmController.js [hunk 2]
```diff
@@ -565,6 +565,10 @@ const userFarmController = {
               .patch({
                 email,
                 status_id: 2,
```
Comment (fsayyid) [inline packages/api/src/controllers/userFarmController.js:567]: Can a pseudo user only converted to a worker? I think the frontend should reflect this if that's the case.

Comment (fsayyid) [reply packages/api/src/controllers/userFarmController.js:567]: I think I understood wrong, I believe you can ignore the above comment.

**Comment by fsayyid** [comment]
When I open the invite link, the gender and info fields are not autofilled in as they should be.

**Review by fsayyid — CHANGES_REQUESTED** [review]
I think you should go through the invite flow after the pseudo user is upgraded, make sure all the info that the FO/FM/EO filled in before inviting the user, is filled in when they click the invite link.

**Comment by fsayyid** [comment]
Also all the info filled in when the pseudo user is first created (phone number, gender, etc) should be filled in the Edit User form when "Convert this worker to a user with an account". And I think I am still unclear on whether a pseudo user can be converted to a FO/FM/EO.

**Comment by seinnlee** [comment]
@fsayyid The invite link should now have the pre-existing information filled out and edit user will also fill out the form (except for "gender", which I haven't figured out why) Gender and Language use the same logic but for some reason Gender isn't filled out when "convert this worker to a user with an account". Maybe you can find the bug while reviewing. Also Kevin says that the pseudo user can be converted to FO/FM/EO.

**Review by fsayyid — APPROVED** [review]
This working well for me now. Good work 👍.

**Review by smattingly — APPROVED** [review]

**Pull request merged**